packages/multi-select-combo-box/test/basic.test.js

```diff
@@ -366,7 +366,7 @@ describe('basic', () => {
       it('should set overflow chip label as not fitting chips count', async () => {
         comboBox.selectedItems = ['apple', 'banana', 'orange'];
         await nextRender();
-        expect(overflow.label).to.equal(2);
+        expect(overflow.label).to.equal('2');
```

Comment:
The property was previously set as number, apparently because of data binding usage.